test/spec/suppressionList.spec.js

```diff
@@ -0,0 +1,136 @@
+var chai = require('chai')
+  , expect = chai.expect
+  , sinon = require('sinon')
+  , sinonChai = require('sinon-chai')
+  , nock = require('nock')
+  , SparkPost = require('../../lib/index');
+
+chai.use(sinonChai);
+
+describe('Suppression List Library', function() {
+  var client, suppressionList;
+
+  before(function () {
+    // setting up a client for all tests to use
+    var key = '12345678901234567890';
+
+    client = new SparkPost(key);
```

Comment:
I would just mock the client since this is a unit test. Then in the below tests all you will need to test is that you are calling the client correctly. Making requests and formatting the response is covered in the client's unit tests.